src/components/AppBar/ImportStudyList.js

```diff
@@ -133,9 +134,11 @@ class ImportStudyList extends PureComponent {
         return (
             <>
                 {/* TODO after mui v5 migration: change icon to ContentPasteGo */}
-                <Button onClick={this.handleOpen} color="inherit" startIcon={<PostAdd />}>
-                    Import
-                </Button>
+                <Tooltip title="Import a schedule from your Study List">
```

Comment:
Looks like your tooltip change snuck in here. I'm gonna be nitpicky here and ask that you undo the change here and raise a separate PR for it. I want to enforce [atomic commits](https://www.aleksandrhovhannisyan.com/blog/atomic-git-commits/) for the repo, and although this is a small change, I want to avoid the [broken window effect](https://storck.io/posts/broken-windows-theory-in-software/)

Reply:
Sorry about that. I've reverted it, which should be ok since you're squashing anyway right?